.github/workflows/update-gh-pages.yaml

```diff
@@ -0,0 +1,26 @@
+name: Update GitHub pages
+
+on:
+  push:
+    tags:
+      # Push events to every tag including hierarchical tags like v1.0/beta
+      - '**'
```

Comment:
should we filter to semver tags only? I feel like everything else would break the update-docs action https://github.com/bakdata/kpops/blob/782d4d9e827b9b3807e422f6da5fe185a87c5530/.github/actions/update-docs/action.yml#L57